spec/setup_acceptance_node.pp

```diff
@@ -1,5 +1,9 @@
-class { 'foreman::repo':
-  repo => 'nightly',
+yumrepo { 'foreman':
+  descr    => "Foreman ${repo}",
```

Comment:
Funny how this (undef variable) freaks out puppet 8 but not 7

Reply:
Puppet 8 has strict variables enabled by default.